core/domain/suggestion_services.py

```diff
@@ -159,7 +159,7 @@ def _update_suggestion(suggestion):
 
     suggestion_model.status = suggestion.status
     suggestion_model.final_reviewer_id = suggestion.final_reviewer_id
-    suggestion_model.change = suggestion.change.to_dict()
+    suggestion_model.change_cmd = suggestion.change.to_dict()
```

Comment:
Nice catch by the way! Good that we now have this under test coverage!

Reply:
Thanks @nithusha21